packages/forma-36-react-components/src/components/Flex/Flex.stories.tsx

```diff
@@ -51,11 +51,12 @@ const DemoBox = ({ times }: { times?: number }) => {
   return <Flex style={styles.demoBox}>Example element</Flex>;
 };
 
-interface Args extends FlexProps {
+interface Args extends FlexOwnProps {
```

Comment:
Do we have to use `Own` here? In the coding style guide we agreed on `ComponentNameProps`

Reply:
We also have `FlexProps` in the component but it includes all polymorphic props and HTML props of the element. `ComponentOwnProps` will be an internal type definition